content/id/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -56,6 +56,7 @@ Sysctl berikut ini didukung dalam kelompok _safe_:
 - `net.ipv4.ip_local_port_range`,
 - `net.ipv4.tcp_syncookies`,
 - `net.ipv4.ping_group_range` (sejak Kubernetes 1.18).
```

Comment:
Optional nit:
(I can see that the period still exists in EN page, but for consistency between unordered list elements:)
(We can also replace this period with comma in EN page:)

```suggestion
- `net.ipv4.ping_group_range` (sejak Kubernetes 1.18),
```

Reply:
done
https://github.com/kubernetes/website/pull/31614